exo-create/features/steps/steps-when.ls

```diff
@@ -10,7 +10,7 @@ require! {
 
 module.exports = ->
 
-  @When /^executing "([^"]*)"$/ (command, done) ->
+  @When /^executing "([^"]*)"$/, timeout: 20_000, (command, done) ->
```

Comment:
Yeah, even when I run just `create-service.feature` it hangs on the same step for the same amount of time. I've `cd`ed into the tmp dir and ran the same command manually and the same behavior exists.